Tests do not work inside container

## error
```
ImportError while loading conftest '/app/tests/conftest.py'.
tests/conftest.py:11: in <module>
    from app.database import get_db
E   ModuleNotFoundError: No module named 'app'
```

## steps to reproduce
- clone repository
- docker compose up
- docker compose exec web pytest

## solution 
add '.' to pytest pythonpath as suggested by [this stackoverflow post](https://stackoverflow.com/questions/49028611/pytest-cannot-find-module)

--- a/.github/workflows/docker-ci.yml
+++ b/.github/workflows/docker-ci.yml
@@ -33,7 +33,7 @@ jobs:
 
       - name: Run tests inside the container
         run: |
-          docker compose exec web python -m pytest
+          docker compose exec web pytest
 
       - name: View logs on failure
         if: failure()

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -30,4 +30,4 @@ jobs:
       - name: Run tests
         shell: bash
         run: |
-          PYTHONPATH="$(pwd)" pytest
+          pytest

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -36,7 +36,7 @@ optional-dependencies.test = [
 
 [tool.pytest.ini_options]
 pythonpath = [
-    ".", "app",
+    "."
 ]
 
 [tool.black]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,6 +34,11 @@ optional-dependencies.test = [
   "sqlalchemy[asyncio]",
 ]
 
+[tool.pytest.ini_options]
+pythonpath = [
+    ".", "app",
+]
+
 [tool.black]
 line-length = 110
 

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,11 +34,6 @@ optional-dependencies.test = [
   "sqlalchemy[asyncio]",
 ]
 
-[tool.pytest.ini_options]
-pythonpath = [
-    "."
-]
-
 [tool.black]
 line-length = 110
 
@@ -47,3 +42,8 @@ profile = "black"
 combine_as_imports = true
 combine_star = true
 line_length = 110
+
+[tool.pytest.ini_options]
+pythonpath = [
+  ".",
+]

